metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/database/SraObjectsImporterThroughDatabase.java

```diff
@@ -190,4 +189,8 @@ private void setEnaObjectQuery(String query) {
         return accessionsToStudy;
     }
 
+    private Map<String, String> getXmls(List<String> accessions) {
```

Comment:
why do we need a list of accessions in parameters?? and slso rename getXmls to getSampleXmls, Good code is self-documenting.

Reply:
Correct, it was made as list for API call and forgot after removing that